tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -149,9 +149,10 @@ describe('Lobby - P0 Perspective', () => {
       assertGameStarted();
     });
   });
-  it.skip('[Missing Feature] Loads lobby after page refresh', () => {
+  // TODO: Rewrite this test when auth is supported in the lobby
+  it.skip('Redirects to game list after page refresh', () => {
```

Comment:
Is this test skipped deliberately? I thought that this feature was in-scope for this PR
```suggestion
  it('Redirects to game list after page refresh', () => {
```